tests/integration/test_integration.py

```diff
@@ -1,21 +1,48 @@
-import os
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
 
-from jina import Flow, Document
+from typing import Callable, List
+import pytest
 
-cur_dir = os.path.dirname(os.path.abspath(__file__))
+from jina import Flow, DocumentArray
+from jinahub.text.encoders.transform_encoder import TransformerTorchEncoder
 
 
-def data_generator(num_docs):
-    for i in range(num_docs):
-        doc = Document(
-            uri=os.path.join(cur_dir, '..', 'data', 'test_image.png'))
-        doc.convert_image_uri_to_blob()
-        yield doc
+@pytest.mark.parametrize(
+    'request_size', [1, 10, 100]
+)
+def test_integration(
+    data_generator: Callable,
+    request_size: int
+):
+    with Flow(return_results=True).add(uses=TransformerTorchEncoder) as flow:
+        resp = flow.post(on='/index', inputs=data_generator(), request_size=request_size)
 
+    for doc in resp[0].docs:
+        assert doc.embedding is not None
 
-def test_use_in_flow():
-    with Flow.load_config('flow.yml') as flow:
-        data = flow.post(on='/index', inputs=data_generator(5))
-        docs = data[0].docs
-        for doc in docs:
-            assert doc.blob.shape == (64, 64, 3)
+
+@pytest.mark.parametrize(
+    ['docs', 'docs_per_path', 'traversal_path'],
+    [
+        (pytest.lazy_fixture('docs_with_text'), [['r', 10], ['c', 0], ['cc', 0]], 'r'),
+        (pytest.lazy_fixture('docs_with_chunk_text'), [['r', 0], ['c', 10], ['cc', 0]], 'c'),
+        (pytest.lazy_fixture('docs_with_chunk_chunk_text'), [['r', 0], ['c', 0], ['cc', 10]], 'cc')
+    ]
+)
+def test_traversal_path(docs: DocumentArray, docs_per_path: List[List[str]], traversal_path: str):
```

Comment:
I think these tests should be part of the `unit` tests.

Reply:
can these be moved to `unit` test? This behavior is better tested without the FLow and independent of it

Reply:
https://github.com/jina-ai/executor-transformer-torch-encoder/pull/1/commits/ab179e069cd325d60e4b337ed8221cf14ae600e7